presets/shared.yml

```diff
@@ -85,6 +85,7 @@ votingKeysDirectory: ./votingkeys
 enableDelegatedHarvestersAutoDetection: true
 catapultAppFolder: /usr/catapult
 rewardProgramAgentPort: 7881
+announceDefaultMaxFee: 100000
```

Comment:
How did you reach that 0.1 as the default max fee?

Reply:
It seems to be a sensible default. Cheap and fast enough.